index.css

```diff
@@ -0,0 +1,199 @@
+body {
+    background-color: var(--dark-green-color);
+    font-size: var(--global-font-size);
+    font-family: Arial, Helvetica, sans-serif;
+    margin: 3.7em 0 0 0 ;
+    padding: 0;
+}
+
+a{
+    text-decoration: none;
+}
+
+
+html {
+    scroll-behavior: smooth;
+}
+
+header.header__navegacion nav ul{
+    position: relative;
+    display: inline;
+    overflow: hidden;
+    position: var(--relative-position);
+    text-align: center;
+    left: 20%;
+    top: 1.3em;
+}
+
+header.header__navegacion nav ul li {
+    list-style: none;
+    display: inline;
+    padding: 0 4em;
+}
+
+header.header__navegacion nav ul li a:hover {
+    color: var(--light-blue-color);
+}
+
+header.header__navegacion nav ul li a {
+    font-weight: var(--bold-font-weight);
+    color: var(--white-color);
+    transition: color .5s;
+}
+
+.boton__citas{
+    float: right;
+    margin: 0.5em 3em;
+    color: var(--dark-green-color);
+    padding: 0.8em 1em;
+}
+
+.boton__citas:hover{
+    background-color: var(--turquoise-background);
+}
+
+.boton__citas:active{
+    -webkit-transform: scale(0.9);
+    -ms-transform: scale(0.9);
+    transform: scale(0.9);
+}
+
+.boton__formulario{
+    margin: 1em auto;
+    padding: 0.8em 1em;
+    border-radius: 0.2em;
+    border: none;
+    color: var(--white-color);
+}
+
+.header__contenido{
+    background-color: var(--cheese-background);
+    height: 31.25em;
+    text-align: center;
+    padding-top: 10rem;
+}
+
+.titulo{
+    font-size: var(--large-font-size);
+    background: -webkit-gradient(linear, left top, right top, color-stop(35%, #ffa500), color-stop(45%, #ff0000), color-stop(60%, purple));
+    background: -o-linear-gradient(left, #ffa500 35%, #ff0000 45%, purple 60%);
```

Comment:
evita propiedades repetidas o tendras errores/ efectos secundarios

Reply:
Hola César, quería explicarte por qué lo hice:
Realmente me estuve documentando para el proyecot, y algunas propiedades CSS como las transiciones, gradientes, y otras, tienen problemas con navegadores antigüos o navegadores con versiones muy viejas. Estás viendo que puse varias veces la misma propiedad porque la propiedad original, que es background-image: linear-gradient(to bottom, var(--dark-green-color) 80%, #003035 100%);, la puse en un generador de prefijos, y se te das cuenta, en las otras ya dice -webkit-gradient, -o-linear-gradient. Esto lo hice con la finalidad de que haya compatibildad en la mayor cantidad de navegadores posibles, y no sólo en unos cuantos por defecto.
Era todo lo que tenía que decirte, pero entonces ya no sé si hice bien o no, en el momento lo creí apropiado.

Reply:
si ese es el caso, entonces esta bien ;). Te felicito por tu investigacion